components/google-cloud/google_cloud_components/aiplatform/remote_runner.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for remote execution of AI Platform component."""
+
+import argparse
+import inspect
+import os
+from google.cloud import aiplatform
+from google.cloud import storage
+
+INIT_KEY = 'init'
+METHOD_KEY = 'method'
+
+
+# TODO(b/184202941) Add type-hinting the functions
+# TODO(b/184203034) Add explanation / exmaples and validation for kwargs
+def split_args(kwargs):
+  """Split args to match MB SDK input format for init, method and config."""
+  init_args = {}
+  method_args = {}
+  config_args = {}
+
+  for key, arg in kwargs.items():
+    if key.startswith(INIT_KEY):
+      init_args[key.split('.')[-1]] = arg
+    elif key.startswith(METHOD_KEY):
+      method_args[key.split('.')[-1]] = arg
+    else:
+      config_args[key.split('.')[-1]] = arg
+
+  return init_args, method_args, config_args
+
+
+def write_to_gcs(project, gcs_uri, text):
```

Comment:
nit: ideally, can you type annotate these arguments, also document them in docstring? 
Not a blocker for this PR, you can address this in a followup PR.

Reply:
Added a TODO(b/184202941) to track this to complete before final release.